### PR TITLE
Created Events and Requests migrations

### PR DESCRIPTION
**Pull request by douglasdgoulart**
# Problem
The project has no migrations for events and requests tables.

# Solution
This migrations were created.

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -8,6 +8,7 @@ play {
     errorHandler = "co.cobli.cobli.block.origin.api.ErrorHandler"
   }
   filters.disabled += play.filters.csrf.CSRFFilter
+  evolutions.db.default.autoApply=true
```
Comment (arielpalmeira) [inline conf/application.conf]:
```suggestion
  evolutions.db.default.autoApply = true
```

File: conf/application.conf [hunk 2]
```diff
@@ -28,4 +29,18 @@ database-execution-context {
...
 }
 
-play.evolutions.db.default.autoApply=true
+
+slick {
+  dbs {
+    default {
+      profile="slick.jdbc.PostgresProfile$"
+      db {
+        driver="org.postgresql.Driver"
```
Comment (arielpalmeira) [inline conf/application.conf]: add whitespaces before and after the `=` signs

File: conf/evolutions/default/1.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+   esn TEXT NOT NULL,
+   desired_state TEXT NOT NULL,
+   device_state TEXT NOT NULL,
+   job_state TEXT NOT NULL,
+   timestamp TEXT NOT NULL,
+   PRIMARY KEY (id)
+);;
+
+# --- !Downs
+DROP TABLE events;;
```
Comment (arielpalmeira) [inline conf/evolutions/default/1.sql]: missing EOFL

File: conf/evolutions/default/2.sql [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+   user_id TEXT NOT NULL,
+   fleet_id TEXT NOT NULL,
+   esn TEXT NOT NULL,
+   desired_state TEXT NOT NULL,
+   timestamp TEXT NOT NULL,
+   PRIMARY KEY (id)
+);;
+
+# --- !Downs
+DROP TABLE requests;;
```
Comment (arielpalmeira) [inline conf/evolutions/default/2.sql]: missing EOFL

File: conf/application.conf [hunk 5]
```diff
@@ -28,4 +29,18 @@ database-execution-context {
   }
 }
 
-play.evolutions.db.default.autoApply=true
+
```
Comment (guilhermeoalima) [inline conf/application.conf]: extra line here

**Comment by guilhermeoalima** [comment]
Where are those tests you were creating?

File: conf/application.conf (hunk 1 above)
Comment (arielpalmeira) [inline conf/application.conf]:
```suggestion
  evolutions.db.default.autoApply = true
```

File: conf/application.conf [hunk 6]
```diff
@@ -28,4 +29,17 @@ database-execution-context {
   }
 }
 
-play.evolutions.db.default.autoApply=true
+slick {
```
Comment (arielpalmeira) [inline conf/application.conf]:
```suggestion
slick.dbs.default {
```

**Pull request merged**